src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java

```diff
@@ -262,6 +266,7 @@ private static boolean isFirstParagraph(DetailNode paragraphTag) {
 
     /**
      * Finds and returns nearest empty line in javadoc.
+     *
```

Comment:
please move this to separate PR to let me merge it immediately and reduce this PR, this will help maintainers to focus on details and nuances of it.

Reply:
Ok. I moved the fixes here: https://github.com/checkstyle/checkstyle/pull/8347 . I will resolve this conversation once that's commit and I rebase this pull request with master.

Reply:
Merged